website/developer-docs/hackathon/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Hackathon 2023
+---
+
+![hackathon-image](./horizontal-brandon-frie-rdHeGGn7rwQ-unsplash.jpg)
+
+## Join us for our first authentik hackathon!
+
+Everyone welcome; we will work on code, docs, and anything else that looks interesting and challenging.
+
+Moderators will be available for most US and European hours, so if during the multi-day event, participants have questions or a PR needs a technical review, we are here for you.
+
+> More details will be added soon about the exact daily timeframes, for both European-based and US-based participants.
+
+Swag? Why, Yes! From cool authentik-branded socks to food certificates, and indoubtibly GitHub fame, we've got you covered.
```

Comment:
```suggestion
Swag? Why, yes! From cool authentik-branded socks to food certificates, and indoubtibly GitHub fame, we've got you covered.
```

Reply:
It's capitalized because it is an answer, but that's not a strict grammar rule. ;-)